packages/ui/src/components/va-date-picker/components/VaDayPicker/VaDayPicker.vue

```diff
@@ -17,6 +22,8 @@
       :key="index"
       @mouseenter="hoveredIndex = index"
       @mouseleave="hoveredIndex = -1"
+      :aria-label="date.toLocaleDateString(undefined, { year: 'numeric', month: 'long', day: 'numeric' })"
```

Comment:
There must be `format` prop, which you can use here.

```suggestion
      :aria-label="$props.format(date)"
```

And probably aria-label must be applied to va-data-picker-cell, which has @click. And most likely date-picker-cell with @click must have `role="button"`.